app/helpers/application_helper.rb

```diff
@@ -0,0 +1,26 @@
+module ApplicationHelper
+  def header(text)
+    content_for(:header) { text.to_s }
+  end
+
+  def sort_link(column, title = nil)
+    title ||= column.titleize
+    direction = column == sort_column && sort_direction == "asc" ? "desc" : "asc"
+    icon = sort_direction == "asc" ? "glyphicon glyphicon-chevron-up" : "glyphicon glyphicon-chevron-down"
+    icon = column == sort_column ? icon : ""
+    link_to "#{title} <span class='#{icon}'></span>".html_safe, {column: column, direction: direction}
```

Comment:
```suggestion
    link_to "#{title} <span class='#{icon}'></span>".html_safe, { column: column, direction: direction }
```
[] donot add spaces at start/end
() donot add spaces at start/end
{} add spaces at start/end